lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -46,9 +46,18 @@ def send(message:, to:)
             region: client_config.config.region,
             extra: response.extra,
           )
-          last_response = response
+        rescue Seahorse::Client::NetworkingError => e
```

Comment:
Voice also rescues `Aws::PinpointSMSVoice::Errors::ServiceError`, but I couldn't find any documentation that suggested SMS behaves the same.